setup.cfg

```diff
@@ -67,3 +67,4 @@ exclude_lines =
   pragma ${PLATFORM_SYSTEM}: no cover
   pragma ${OS_NAME}: no cover
   pragma Python [0-9.,]*${PYTHON_VERSION}[0-9.,]*: no cover
+  sys.exit\(main\(\)\)
```

Comment:
We can have a alternate way, check this out https://medium.com/@george.shuklin/how-to-test-if-name-main-1928367290cb

Reply:
I've explained on https://gitlab.com/coala/mobans/issues/84 why I think that is unnecessary.

If necessary, we have a check script which verifies that all entry points defined in the yaml have exactly these two lines at end of the file.  That prevents these sequences from being different in multiple entry points, and also ensures they are identical on all repos with entry points.